scripts/commerce.js

```diff
@@ -151,20 +151,28 @@ query($sku: String!) {
 }
 `;
 
+export async function commerceEndpointWithQueryParams() {
+  // Set Query Parameters so they can be appended to the endpoint
+  const queryParameters = new URLSearchParams();
+  queryParameters.append('Magento-Environment-Id', await getConfigValue('commerce-environment-id'));
+  queryParameters.append('Magento-Website-Code', await getConfigValue('commerce-website-code'));
+  queryParameters.append('Magento-Store-View-Code', await getConfigValue('commerce-store-view-code'));
+  queryParameters.append('Magento-Store-Code', await getConfigValue('commerce-store-code'));
+  queryParameters.append('Magento-Customer-Group', await getConfigValue('commerce-customer-group'));
+  const urlWithQueryParams = new URL(await getConfigValue('commerce-endpoint'));
```

Comment:
Please make sure to not remove any already existing query parameters in the `commerce-endpoint`. This for example happens when using API mesh. So it's better to append the values.

Reply:
i refactored the code so that each value is appended directly to searchParams